resources/css/database.css

```diff
@@ -0,0 +1,63 @@
+body{
+	padding: 10px 30px;
+}
+
+.pagetitle {
+	width: 506px;
+	height: 50px;
```

Comment:
Specifying the width and height for .pagetitle results in the h1 header on the database page to overlap with the p tag below it. These specified width and height values should be removed (unless they are made larger so that there is no overlap, but I think that removing the values altogether would be best)